internal/entfilter/filter.go

```diff
@@ -7,7 +7,8 @@ import (
 	"github.com/google/uuid"
 	"golang.org/x/exp/maps"
 
-	"github.com/artefactual-sdps/enduro/internal/enums"
+	ingest_enums "github.com/artefactual-sdps/enduro/internal/enums"
```

Comment:
At some point I'd like to have a similar directory structure for the ingest and storage packages and sub-packages.